rollup.config.js

```diff
@@ -47,6 +51,36 @@ const getUMDConfig = ({ env, file }) => {
 };
 
 export default [
-  getUMDConfig({ env: 'development', file: 'dist/react-beautiful-dnd.js' }),
+  getUMDConfig({ env: 'development', file: 'dist/react-beautiful-dnd.umd.js' }),
+
   getUMDConfig({ env: 'production', file: 'dist/react-beautiful-dnd.min.js' }),
```

Comment:
.umd.min.js?

Reply:
This will break pattern :(

```
react-beautiful-dnd.cjs.js
react-beautiful-dnd.cjs.js.flow
react-beautiful-dnd.esm.js
react-beautiful-dnd.min.js
react-beautiful-dnd.umd.js
```

I just would like to see this pattern widespread, so `min` in most cases could mean `umd` format. I already used it in a set of modules.